frontend/src/components/notification/Notification.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+class Notification extends Component {
+  state = {
+response: [],
+deletedId: 0
+  }
+
+
+handleFriendRequestStatus(id, url){
```

Comment:
how did you want to handle the response if it was 200 or 404 or something like that,
before each catch you have to make .then()